src/inet.c

```diff
@@ -49,6 +49,9 @@ int uv_inet_ntop(int af, const void* src, char* dst, size_t size) {
   /* NOTREACHED */
 }
 
+char* uv_if_indextoname(unsigned int ifindex, char* ifname) {
+    return if_indextoname(ifindex, ifname);
```

Comment:
@santigimeno I can do this (at least the first version), but I have difficulty in figuring out in which file(s) I should place the Unix and Windows versions.  Could you please suggest suitable source files for the source code?

--Pekka